### PR TITLE
sql statement: Add syntax gram of `load stats`

### PR DESCRIPTION
**Pull request by qw4990**
### What is changed, added or deleted? (Required)

Add syntax gram of `load stats`.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3362

### CODE REVIEW

File: sql-statements/sql-statement-load-stats.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (yikeke) [inline sql-statements/sql-statement-load-stats.md:1]: Please add this file to TOC.md. @qw4990

**Comment by yikeke** [comment]
This PR is aligned from https://github.com/pingcap/docs-cn/pull/3362? If so, you also need to add the `needs-cherry-pick-2.1` label. @qw4990

File: sql-statements/sql-statement-load-stats.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+title: LOAD STATS
+summary: An overview of the usage of LOAD STATS for the TiDB database.
+category: reference
+---
+
+# LOAD STATS
+
+The `LOAD STATS` statement is used to load statistics data into TiDB.
```
Comment (yikeke) [inline sql-statements/sql-statement-load-stats.md:9]:
```suggestion
The `LOAD STATS` statement is used to load the statistics into TiDB.
```

File: sql-statements/sql-statement-load-stats.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Synopsis
+
+**LoadStatsStmt:**
+
+![LoadStatsStmt](/media/sqlgram/LoadStatsStmt.png)
+
+## Examples
+
+Users can access the address `http://${tidb-server-ip}:${tidb-server-status-port}/stats/dump/${db_name}/${table_name}` to download the TiDB instance's statistics data.
```
Comment (yikeke) [inline sql-statements/sql-statement-load-stats.md:19]:
```suggestion
You can access the address `http://${tidb-server-ip}:${tidb-server-status-port}/stats/dump/${db_name}/${table_name}` to download the TiDB instance's statistics.
```

File: sql-statements/sql-statement-load-stats.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Users can use `LOAD STATS ${stats_path}` to load the specific statistics file.
```
Comment (yikeke) [inline sql-statements/sql-statement-load-stats.md:21]:
```suggestion
You can also use `LOAD STATS ${stats_path}` to load the specific statistics file.
```

File: sql-statements/sql-statement-load-stats.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+The `${stats_path}` can be an absolute path or a relative path, and here is an example:
```
Comment (yikeke) [inline sql-statements/sql-statement-load-stats.md:23]:
```suggestion
The `${stats_path}` can be an absolute path or a relative path. If you use a relative path, the corresponding file is found starting from the path where `tidb-server` is started. Here is an example:
```

**Comment by qw4990** [comment]
All comments have been addressed, PTAL @yikeke

**Pull request merged**